docs/docs/tutorial/chapter5/testing.md

```diff
@@ -12,16 +12,47 @@ Since we just started the suite, and we haven't changed any files yet, it may no
 
 ![tests_running](https://user-images.githubusercontent.com/46945607/165376937-89ed9254-0d8e-4945-a0d9-17178764a4b0.png)
 
-If you cloned the example repo during the intermission and followed along with the Storybook tutorial in this chapter, the test run should finish and you will see something like this: 
+If you cloned the example repo during the intermission and followed along with the Storybook tutorial in this chapter, the test run should finish and you will see something like this:
 
 ![suite_finished](https://user-images.githubusercontent.com/46945607/165378519-2859dd0d-d46a-448f-a62e-0b8f91c55a87.png)
 
-Note that the summary on the bottom indicates that there was 1 test that failed. If you feel curious, you can scroll up in your terminal and see more details on the test that failed. We'll also take a look at that failed test shortly. 
+:::info
+
+If you decided to keep your codebase from the first part of the tutorial, then you will get the following error after running
+
+```bash
+yarn rw test
+```
+
+```bash
+Error: Get config: Schema Parsing P1012
+
+error: Error validating datasource `db`: the URL must start with the protocol `postgresql://` or `postgres://`.
+  -->  schema.prisma:3
+   |
+ 2 |   provider = "postgresql"
+ 3 |   url      = env("DATABASE_URL")
+   |
+
+Validation Error Count: 1
+
+error Command failed with exit code 1.
+```
```

Comment:
- it feels like this could all be one codeblock
- tweak the copy slightly (just shortened it)

```suggestion
```bash
yarn rw test

# ...

Error: Get config: Schema Parsing P1012

error: Error validating datasource `db`: the URL must start with the protocol `postgresql://` or `postgres://`.
  -->  schema.prisma:3
   |
 2 |   provider = "postgresql"
 3 |   url      = env("DATABASE_URL")
   |
```
```